R/lockfile-api.R

```diff
@@ -0,0 +1,141 @@
+
+# NOTE: These functions are used by the 'dockerfiler' package, even though
```

Comment:
File an issue on dockerfiler and link to it here? Or file issue in renv for us to do later?

Reply:
Filed here: https://github.com/ThinkR-open/dockerfiler/issues/56